Sources/Gtk/Utility/GObjectProperty.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Copyright © 2016 Tomas Linhart. All rights reserved.
+//
+
+import CGtk
+
+@propertyWrapper public struct GObjectProperty<Value: GValueRepresentable> {
+    @available(*, unavailable)
+    public var wrappedValue: Value {
+        get { fatalError("This wrapper only works on instance properties of GObjectRepresentable") }
+        set { fatalError("This wrapper only works on instance properties of GObjectRepresentable") }
+    }
+
+    var name: String
+
+    public init(named: String) {
+        self.name = named
+    }
+
+    public static subscript<EnclosingSelf: GObjectRepresentable>(
+        _enclosingInstance instance: EnclosingSelf,
+        wrapped _: ReferenceWritableKeyPath<EnclosingSelf, Value>,
+        storage storageKeyPath: ReferenceWritableKeyPath<EnclosingSelf, Self>
+    ) -> Value {
```

Comment:
Woah super cool! I didn't know you could access the enclosing instance like this

Reply:
Yep! Is how Published property wrapper work without mirror, we could use it for our Observed